.flake8

```diff
@@ -1,2 +1,3 @@
 [flake8]
 max-line-length = 120
+ignore = E402  # E402 = Module level import not at top of file (due to monkey patching)
```

Comment:
Is https://github.com/bbc/nmos-common/blob/master/nmoscommon/query.py#L22-L30 the only place this matters? Would it be better specified with `# noqa E402` on each of those lines instead of turning the rule off?